environment.py

```diff
@@ -418,5 +418,8 @@ def env() -> Mapping[str, Optional[str]]:
         'azul_velocity_zenhub_pkey': None,
 
         # URL of the ZenHub workspace with all repos selected
-        'azul_velocity_zenhub_url': None
+        'azul_velocity_zenhub_url': None,
+
+        # Path to tracked Terraform schema used to generate *.tf.json files.
+        'AZUL_TRACKED_SCHEMA_PATH': '{project_root}/terraform/_schema.json'
```

Comment:
There is no point in making this configurable. It should be hard-coded in Config. At least that was my intention when we discussed this at stand-up.